src/libraries/System.Private.CoreLib/src/System/Version.cs

```diff
@@ -85,8 +88,8 @@ public Version(string version)
 
         public Version()
         {
-            _Major = 0;
```

Comment:
For clarity, I would keep `_Major = 0;` and `_Minor = 0;` here, but comment them out.